src/App.js

```diff
@@ -22,6 +24,8 @@ function App() {
 			<Route path="/" element={<ProtectedRoute />}>
 				<Route index element={<Dashboard />} />
 				<Route path="dashboard" element={<Dashboard />} />
+				<Route path="quest" element={<Quest />} />
+				<Route path="questTwo" element={<Quest2 />} />
```

Comment:
Little detail: better practice is to use kebab case for paths, so 'quest-two'. Ideally, "quest" should be named "quests", since the path contains a list of all quests.